addons/mail/models/discuss/discuss_channel.py

```diff
@@ -68,7 +68,7 @@ def _generate_random_token(self):
     sfu_channel_uuid = fields.Char(groups="base.group_system")
     sfu_server_url = fields.Char(groups="base.group_system")
     rtc_session_ids = fields.One2many('discuss.channel.rtc.session', 'channel_id', groups="base.group_system")
-    last_call_message_id = fields.Many2one("mail.message", string="Message of last call start")
+    call_message_ids = fields.One2many("mail.message", "call_channel_id", string="Messages of call start")
```

Comment:
We don't want to store `call_channel_id` on mail message.
I guess what should be done instead is to make this a many2many so it has its own table.